DFC.ServiceTaxonomy.Neo4j/Synonyms/src/main/java/analyzer/SynonymHelper.java

```diff
@@ -0,0 +1,61 @@
+import org.apache.lucene.analysis.Analyzer;
+import org.apache.lucene.analysis.core.LowerCaseFilterFactory;
+import org.apache.lucene.analysis.custom.CustomAnalyzer;
+import org.apache.lucene.analysis.standard.StandardFilterFactory;
+import org.apache.lucene.analysis.standard.StandardTokenizerFactory;
+import org.apache.lucene.analysis.synonym.SynonymFilterFactory;
+import java.io.*;
+import org.apache.commons.io.FileUtils;
+import java.util.Scanner;
+
+public class SynonymHelper {
+
+    public SynonymHelper() {
```

Comment:
Don't think you need to supply an empty constructor.

Reply:
I'll give it a go - it was complaining it couldn't construct